Workbooks/Azure Security Center/Compliance/Compliance.workbook

```diff
@@ -1632,7 +1632,7 @@
                 {
                   "type": 1,
                   "content": {
-                    "json": "To use this Compliance Over Time workbook, you'll need to configure **continuous export** to export data to a Log Analytics workspace:\r\n1. From Security Center's sidebar, select **Pricing & settings**.\r\n2. Select the specific subscription for which you want to configure the data export.\r\n3. From the sidebar of the settings page for that subscription, select **Continuous Export**.\r\n4. Set the export targed to **Log Analytics workspace**.\r\n5. Select the following data types: **Regulatory compliance**.\r\n6. From the export frequency options, select **Streaming** and **Snapshots (Preview)**.\r\n7. Select **Save**.\r\n8. Wait for the first snapshot to occur.\r\n\r\n[Learn more](https://docs.microsoft.com/en-us/azure/security-center/continuous-export?tabs=azure-portal#set-up-a-continuous-export)\r\n\r\n> **Notes**\r\n* To get full visibility, wait at least one week for the first snapshot to be exported.\r\n* To configure continuous export across your organization, use the supplied Azure Policy 'DeployIfNotExist' policies described [here](https://docs.microsoft.com/en-us/azure/security-center/continuous-export?tabs=azure-policy#set-up-a-continuous-export)."
+                    "json": "To use this Compliance Over Time workbook, you'll need to configure **continuous export** to export data to a Log Analytics workspace:\r\n1. From Defender for Cloud sidebar, select **Environment settings**.\r\n2. Select the specific subscription for which you want to configure the data export.\r\n3. From the sidebar of the settings page for that subscription, select **Continuous Export**.\r\n4. Set the export target to **Log Analytics workspace**.\r\n5. Select the following data type: **Regulatory compliance**.\r\n6. From the export frequency options, select **Streaming updates** and **Snapshots (Preview)**.\r\n7. Select **Save**.\r\n8. Wait for the first snapshot to occur.\r\n\r\n[Learn more](https://docs.microsoft.com/en-us/azure/security-center/continuous-export?tabs=azure-portal#set-up-a-continuous-export)\r\n\r\n> **Notes**\r\n* To get full visibility, wait at least one week for the first snapshot to be exported.\r\n* To configure continuous export across your organization, use the supplied Azure Policy 'DeployIfNotExist' policies described [here](https://docs.microsoft.com/en-us/azure/security-center/continuous-export?tabs=azure-policy#set-up-a-continuous-export)."

```

Comment:
The links should get an update as well. 

The first one:
https://docs.microsoft.com/en-us/azure/defender-for-cloud/continuous-export?tabs=azure-portal#set-up-a-continuous-export

The second one:
https://docs.microsoft.com/en-us/azure/defender-for-cloud/continuous-export?tabs=azure-policy#set-up-a-continuous-export

Are we still considering this workbook as preview?

Reply:
Changed the links.
It's still in preview. I'm not sure if we should add more features in the workbook before GA but it's not top priority right now.